src/gtest/test_mempoollimit.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2019 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php 
+
+#include <gtest/gtest.h>
+#include <iostream>
+
+#include "arith_uint256.h"
+#include "mempoollimit.h"
+#include "utiltime.h"
+#include "utiltest.h"
+#include "transaction_builder.h"
+
+
+const uint256 TX_ID1 = ArithToUint256(1);
+const uint256 TX_ID2 = ArithToUint256(2);
+const uint256 TX_ID3 = ArithToUint256(3);
+
+TEST(MempoolLimitTests, RecentlyEvictedList_AddWrapsAfterMaxSize)
```

Comment:
https://github.com/google/googletest/blob/master/googletest/docs/faq.md#why-should-test-suite-names-and-test-names-not-contain-underscore